src/main.js

```diff
@@ -209,7 +209,6 @@ function createSavedPostersSection() {
 };
 
 function deleteThisPoster(event) {
-  console.log(event.target.id);
   var id = event.target.id;
   for (var i = 0; i < savedPosters.length; i++) {
     console.log(typeof id, typeof savedPosters[i].id);
```

Comment:
Don't forget to remove this console.log as well.

Reply:
Thanks I will update and re-do the pull request.